NVIDIA/benchmarks/maskrcnn/implementations/pytorch/maskrcnn_benchmark/data/build.py

```diff
@@ -99,19 +99,19 @@ def _compute_aspect_ratios(dataset):
 
 
 def make_batch_data_sampler(
-    dataset, sampler, aspect_grouping, images_per_batch, num_iters=None, start_iter=0, random_number_generator=None,
+    dataset, sampler, aspect_grouping, images_per_batch, num_iters=None, start_iter=0, random_number_generator=None, is_train=True
 ):
     if aspect_grouping:
         if not isinstance(aspect_grouping, (list, tuple)):
             aspect_grouping = [aspect_grouping]
         aspect_ratios = _compute_aspect_ratios(dataset)
         group_ids = _quantize(aspect_ratios, aspect_grouping)
         batch_sampler = samplers.GroupedBatchSampler(
-            sampler, group_ids, images_per_batch, drop_uneven=False
+            sampler, group_ids, images_per_batch, drop_uneven=is_train
         )
     else:
         batch_sampler = torch.utils.data.sampler.BatchSampler(
-            sampler, images_per_batch, drop_last=False
+            sampler, images_per_batch, drop_last=is_train
```

Comment:
Why don't we need this in eval?

Reply:
COCO eval requires the entire eval dataset.
I did not encounter the batch can not be divided by microbatch issue in eval.

Reply:
ok